src/main/java/seedu/address/logic/parser/AddPersonCommandParser.java

```diff
@@ -45,9 +46,10 @@ public AddPersonCommand parse(String args, Model model) throws ParseException {
             Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE)).get();
             Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL)).get();
             Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS)).get();
+            Amount amount = new Amount("0.00");
             Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
-            Person person = new Person(name, phone, email, address, tagList);
+            Person person = new Person(name, phone, email, address, amount, tagList);
 
```

Comment:
Same here, I think what you called Amount, should be called "Balance" and should be instanciated at 0$. Because when a person is added to the addressbook, he isn't implied in any transaction, so he does not owe anyone.

Reply:
Yes it is instanciated already at 0 dollar when each person is added. Okay will change from amount to balance.